website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -31,14 +32,18 @@ After you link your HCP and Terraform Cloud accounts, you can only log in with y
 
 You cannot unlink an autogenerated Terraform Cloud account from your HCP account. You can unlink a pre-existing Terraform Cloud account on the [HCP Account Linking page](#hcp-account-linking) in your **User settings**.
 
+<!-- END: TFC:only -->
+
 ## Creating an Account
 
 To use Terraform Cloud or Enterprise, you must create an account through one of the following methods:
 
 - **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
```

Comment:
```suggestion
- **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you can automatically join that organization and can begin using Terraform Cloud.
```
Suggest changing "will" to "can" to avoid future tense.